betterconf/caster.py

```diff
@@ -11,15 +11,18 @@ def cast(self, val: str):
 
 class ConstantCaster(AbstractCaster, typing.Generic[VT]):
 
-    ABLE_TO_CAST: typing.Dict[str, typing.Any] = {}
+    ABLE_TO_CAST: typing.Dict[typing.Union[str, typing.Tuple[str]], typing.Any] = {}
 
     def cast(self, val: str) -> typing.Union[str, VT]:
         """Cast using ABLE_TO_CAST dictionary as in BoolCaster"""
-        converted = self.ABLE_TO_CAST.get(val.lower())
-        if not converted:
-            return val
-        else:
+        if val in self.ABLE_TO_CAST.keys():
+            converted = self.ABLE_TO_CAST.get(val.lower())
             return converted
+        else:
+            for key in self.ABLE_TO_CAST.keys():
+                if isinstance(key, tuple) and val.lower() in key:
```

Comment:
doesn't it support cases when key is not a tuple? 'cause, it seems to me so.

Reply:
Now supports.